ports/oatpp-openssl/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "oatpp-openssl",
+  "version": "1.2.5",
```

```suggestion
  "version-semver": "1.2.5",
```
It would be better severed by [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver) here.

---

done ;-)